simde/x86/svml.h

```diff
@@ -4370,6 +4370,138 @@ simde_mm_erfinv_ps (simde__m128 a) {
   #define _mm_erfinv_ps(a) simde_mm_erfinv_ps(a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128d
+simde_mm_erfinv_pd (simde__m128d a) {
+  #if defined(SIMDE_X86_SVML_NATIVE) && defined(SIMDE_X86_SSE_NATIVE)
+    return _mm_erfinv_pd(a);
+  #elif defined(SIMDE_ASSUME_VECTORIZATION)
+    simde__m128d one = simde_mm_set1_pd(1.0f);
```

Comment:
These should be `SIMDE_FLOA64_C(1.0)`; 1.0f is a 32-bit float.  The rest looks good to me, so I'll just tweak this when I merge it.